src/hooks/useEtherspotHistory.ts

```diff
@@ -59,24 +64,47 @@ const useEtherspotHistory = (chainId?: number): IEtherspotHistoryHook => {
 
   const getAccountTransaction = async (
     hash: string,
-    historyChainId: number = defaultChainId,
+    historyChainId: number = defaultChainId
   ): Promise<Transaction | undefined> => {
     try {
       const dataService = getDataService();
+      console.log('dataservice getAccountTransaction', dataService);
```

Comment:
Ooops, no informational console logs here please (it pollutes the developer's developer console)

Reply:
I think i added this actually and forgot to remove it, apologies

Reply:
Oops good catch!